src/components/account-header/account-header.yaml

```diff
@@ -59,7 +59,6 @@ examples:
           href: '/components/account-header/with-service-name/preview'
         cy:
           href: '/components/account-header/welsh-with-service-name/preview'
-      serviceName: 'My service'
```

Comment:
This key was a duplicate

Reply:
Thanks, I've been meaning to remove that for about a year :)